connect-examples/v2/node_gift-cards/README.md

```diff
@@ -1,309 +1,359 @@
-# gift-card-api-sample-app
+# Useful Links
 
-## **Overview**
+- [Node.js SDK Page](https://developer.squareup.com/docs/sdks/nodejs)
+- [Gift Cards API Overview](https://developer.squareup.com/docs/gift-cards/using-gift-cards-api)
+- [Gift Cards API Reference](https://developer.squareup.com/reference/square/gift-cards-api)
+- [Gift Card Activities API Reference](https://developer.squareup.com/reference/square/gift-card-activities-api)
 
-This sample web application integrates the Square [Gift Cards API](https://developer.squareup.com/reference/square/gift-cards-api) and showcases its functionality, including:
+# Gift Card API Sample App
 
-*   Creating a gift card.
-*   Activating a gift card.
-*   Linking and unlinking a gift card to and from a customer. 
-*   Adding funds to a gift card.
-*   Viewing gift card activities.
+- [Overview](#overview)
+- [Test data](#test-data)
+- [Setup](#setup)
```

Comment:
setup should come before test data per the actual order of things in this README